README.md

```diff
@@ -1,2 +1,22 @@
 # hw01
 Repository for homework 01
+
+# This is a brief description of Elsy Gonzalez
+
+My name is Elsy Gonzalez, I am a third year Ph.D student in **Political Science** with a specialization in *International Relations*.
+
+I have a BA degree in Political Science and International Relations from the *Centro de Investigación y Docencia Economicas* in Mexico City, Mexico.
+
+## Interests
+
+My substantive interests are the following:
+
+1. International recognition of statehood
+
+2. Recognition of secessionist and irredentist movements
+
+3. Diaspora lobbying
+
+    a. I am specifically interested in diaspora lobbying efforts for the purpose of garnering recognition for the independence of a secessionist/irredentist movement.
+    
+For more information, please see my description on the **UChicago** *Political Science* website at https://political-science.uchicago.edu/directory/elsy-gonzalez
```

Comment:
To explicitly add a URL but have the link actually be meaningful text, use the syntax `[]()`.

```
For more information, please see my description on the [**UChicago** *Political Science* website](https://political-science.uchicago.edu/directory/elsy-gonzalez)
```

For more information, please see my description on the [**UChicago** *Political Science* website](https://political-science.uchicago.edu/directory/elsy-gonzalez)